gameBase/src/css/styles.css

```diff
@@ -30,6 +30,14 @@ body {
   transition: transform 0.5s;
 }
 
+.memory-card img:hover {
```

Comment:
nice effect just can you adapted so only when the card is not flip have that hover effect, hint on the class flip remove the hover effect